datafusion/physical-expr/src/utils/guarantee.rs

```diff
@@ -867,13 +868,6 @@ mod test {
         LiteralGuarantee::try_new(column, Guarantee::NotIn, literals.iter()).unwrap()
     }
 
-    /// Convert a logical expression to a physical expression (without any simplification, etc)
-    fn logical2physical(expr: &Expr, schema: &Schema) -> Arc<dyn PhysicalExpr> {
```

Comment:
I was going to suggest using `SessonState::create_physical_expr` or `SessionContext::create_physical_expr` but I think that those are not available to code in the physical-expr crate

https://github.com/apache/datafusion/blob/b468ba788310d5dcd6ea4549f0d5b0f215ea49c9/datafusion-examples/examples/expr_api.rs#L126